### DISCUSSION TITLE
Renovate keeps timing out

### DISCUSSION THREAD

**Discussion by lucascurti** [renovatebot/renovate#19029] [Request Help]
We are setting up renovate on our repo using the hosted Github app. After pushing a config update, renovate started timing out:
```
DEBUG: Using docker to execute(branch="renovate/mime-3.x")
{
  "image": "sidecar"
}
DEBUG: Resolved stable matching version(branch="renovate/mime-3.x")
{
  "toolName": "node",
  "constraint": null,
  "resolvedVersion": "v18.12.1"
}
DEBUG: containerbaseDir is separate from cacheDir(branch="renovate/mime-3.x")
DEBUG: No tag or tagConstraint specified for image: docker.io/renovate/sidecar(branch="renovate/mime-3.x")
DEBUG: Docker image is already prefetched: docker.io/renovate/sidecar@sha256:5103d041fbc1e3deb7fd201ad0ed6ade56dc5c7f44c1613b094e58c022167fe8(branch="renovate/mime-3.x")
WARN: Job timeout(branch="renovate/mime-3.x")
```
It always fails at the same point: `Docker image is already prefetched`.

I've tried [nuking the config](https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard) as instructed in the docs but the problem persists. Any idea what might be going on? Could this be a cache issue?

**Comment by rarkins** [comment]
This indicates that a `docker run` command has timed out while processing the branch `renovate/mime-3.x`. It could be due to a bug (including npm) but also due to the fact the repo takes too long to process. Is this the first branch it processes or one of many? You could try reducing concurrentPrLimit to a lower value <5 to increase the chance that Renovate can complete each job

**Reply by lucascurti** [reply]
Thanks! I'll try that. It's one of many.
